### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by Mutai-Gilbert**
In this pull request we were able to:
- Understand how to use medium-fidelity wireframes to create a UI.
-  Understand different ways to create objects in JavaScript.
- Create and access properties and methods of JavaScript objects.

### CODE REVIEW

File: modules/store.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(id) {
+    let books = Store.getBooks();
+    id = parseInt(id, 10);
+    books = books.filter((book) => book.id !== id);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
```
Comment (codepantha) [inline modules/store.js:23]: - [x] I noticed a small bug 🐛 where if you delete a book, its element gets removed from the DOM but the book still exists in `localStorage` so when you reload the page, the book appears again. Please try to fix this in order to get approved 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Data preserved in localStorage 📌 
✔️ Used GitHub Flow
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please consider adding a license to your project as it is advisable for every software (opensource included) to have a license 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,137 @@
...
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Project from [Microverse](https://www.microverse.org/?grsf=i6yi2m) html, css & Javascript module
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (jerryowusu) [inline README.md:135]: [OPTIONAL] Kindly fix the link to the license since it is not working properly. :pray:

**Review by jerryowusu — APPROVED** [review]
Hi  Team :wave: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

kpop-approved


## Highlights
All linters are working well :100: 
Well documented readme :+1: 
Used GitHub Flow :1st_place_medal: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bondok6** [comment]
## Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, which is already approved, this project is a [pair-programming project](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md#:~:text=Work%20and%20submission%20mode,of%20your%20pair%2Dprogramming%20group.) so only one student should request a review.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**